code/datums/supplypacks/contraband.dm

```diff
@@ -52,4 +52,63 @@
  	cost = 50
  	contraband = 1
  	containertype = /obj/structure/closet/crate/secure/weapon
- 	containername = "Weapons crate"
\ No newline at end of file
+ 	containername = "Weapons crate"
+
+/datum/supply_packs/randomised/misc/telecrate //you get something awesome, a couple of decent things, and a few weak/filler things
+	name = "ERR_NULL_ENTRY"
+	num_contained = 1
+	contains = list(
```

Comment:
Seems like the lists within this list are indented too far.

Reply:
DM still hates me

Reply:
I was talking about this: ![](https://image.prntscr.com/image/yl5Jr6KwR6_F3dYPRQqwqA.png)